scripts/install/usr/libexec/cortx-motr/motr-start

```diff
@@ -32,6 +32,7 @@ _handle_sigterm() {
    if [[ -z $PID ]]; then
       echo "No m0d process to kill"
    else
+      kill -SIGTERM $PID 2> /dev/null
```

Comment:
Is there any way to know which Signal is received and pass only that signal here ?

Reply:
Added 2 separate functions for each signal handlers